plaso/parsers/mac_wifi.py

```diff
@@ -229,14 +229,16 @@ def _ParseLogLine(self, parser_mediator, key, structure):
 
     function = self._GetValueFromStructure(structure, 'function')
     if function:
-      # Due to the use of CharsNotIn the function value can contain leading
```

Comment:
why remove this?

Reply:
CharsNotIn is not used in the function definition. Removed the strip here altogether, as it's not necessary.

Reply:
It is used in the pyparsing grammer, https://github.com/log2timeline/plaso/blob/e5808cb4b35104b00db2f344d3c6a036e328b000/plaso/parsers/mac_wifi.py#L69

Reply:
per conversation, this applies to agent not function and text.